flash_tutorials/tabular_classification/.meta.yml

```
@@ -0,0 +1,20 @@
+title: Tabular Classification
+author: Pablo Moreno (pabloantoniomorenocasares@gmail.com)
+created: 2021-12-12
+updated: 2021-12-18
+license: CC BY-SA
+build: 1
+tags:
+  - Tabular
+  - Classification
+description: |
+  This tutorial covers using Lightning Flash to predict whether some users will make a transaction based on numerical tabular data. Since it is an inbalanced dataset, we use SMOTE to balance the dataset. We then use native TabNet to train the model.
+requirements:
+  - pandas==1.1.5
+  - lightning-flash[tabular]>=0.6.0
+  - torchmetrics
+  - imblearn
+  - sklearn
+accelerator:
+  - GPU
+  - CPU
```

```suggestion
  - CPU
datasets:
  kaggle:
    - santander-customer-transaction-prediction
```
missing data: https://www.kaggle.com/c/santander-customer-transaction-prediction